tests/test_browser.py

```diff
@@ -5304,15 +5304,20 @@ def test(args):
     # thread and busy-wait
     'main_thread': (['-sPTHREAD_POOL_SIZE=1'],),
     # using proxy_to_pthread also works, of course
-    'proxy_to_pthread': (['-sPROXY_TO_PTHREAD'],),
+    'proxy_to_pthread': (['-sPROXY_TO_PTHREAD', '-sINITIAL_MEMORY=32MB', '-DPROXYING'],),
```

Comment:
Does the test really need 32mb or memory?  I guess this is because the per-thread overhead in emscripten is so high?

Reply:
Yes. This is because it needs memory for  all the threads. Now it uses a thread per backend.
I can set the memory to the exact minimum needed by the test.

Reply:
Keeping it at a largish round number like 32MB is better than trying to find the exact number since it will make the test robust against unrelated increases in memory consumption.